integration-test/1491-norwegian-shields.py

```diff
@@ -0,0 +1,135 @@
+# -*- encoding: utf-8 -*-
+from . import FixtureTest
+
+
+class NorwegianShieldTest(FixtureTest):
+    def test_no_oslo_ring(self):
+        # there's a different sign for the Store Ringvei (Oslo Ring-road). also
+        # looks like in Norway they actually sign the E-roads.
+        #
+        # https://en.wikipedia.org/wiki/Ring_3_(Oslo)
+        # https://en.wikipedia.org/wiki/Norwegian_national_road
+
+        import dsl
+
+        z, x, y = (16, 34736, 19062)
+
+        self.generate_fixtures(
+            dsl.is_in('NO', z, x, y),
+            # https://www.openstreetmap.org/way/71218142
+            dsl.way(71218142, dsl.tile_diagonal(z, x, y), {
+                'highway': u'trunk',
+                'int_ref': u'E 06',
+                'lanes': u'2',
+                'maxspeed': u'70',
+                'maxspeed:conditional': u'60 @ (Nov 01 - Apr 8)',
+                'name': u'Adolf Hedins vei',
+                'oneway': u'yes',
+                'ref': u'E 6;Ring 3',
+                'source': u'openstreetmap.org',
+            }),
+            dsl.relation(1, {
+                'nat_ref': u'E 6',
+                'network': u'e-road',
+                'ref': u'E 06',
+                'route': u'road',
+                'section': u'Norway (south-south)',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikidata': u'Q921422',
+                'wikipedia': u'de:Europastraße 6',
+            }, ways=[71218142]),
+            dsl.relation(2, {
+                'name': u'Store ringvei',
+                'ref': u'Ring 3',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[71218142]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 71218142,
+                'network': u'NO:oslo:ring',
```

Comment:
Do all Norway admin-1 regions get region names like this? If so we need to enumerate them for client-side perf as a comment here.

Reply:
No, it's only the [Oslo ring road](https://en.wikipedia.org/wiki/Ring_3_(Oslo)).

The full set of `network` values output in Norway should be:

* `NO:Riksvei`
* `NO:Fylkesvei`
* `NO:oslo:ring`
* `e-road`